packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -416,6 +416,22 @@ export abstract class RestApiBase extends Resource implements IRestApi {
       }
     }
   }
+
+  protected configureEndpoints(props: RestApiProps): CfnRestApi.EndpointConfigurationProperty | undefined {
```

Comment:
Mark this as an internal API

```suggestion
  /**
   * @internal
   */
  protected configureEndpoints(props: RestApiProps): CfnRestApi.EndpointConfigurationProperty | undefined {
```

Reply:
Addressed in https://github.com/aws/aws-cdk/pull/9068/commits/7a097a16e6aac372d2485790807310ef9df8633a